tsdb/engine.go

```diff
@@ -78,7 +78,7 @@ type Engine interface {
 	Statistics(tags map[string]string) []models.Statistic
 	LastModified() time.Time
 	DiskSize() int64
-	IsIdle() bool
+	IsIdle(isLogged bool) bool
```

Comment:
Would it be better to do something like `IsIdle() (state bool, reason string)` where:
* `state == false` & `reason` describes why
* `state == true` & `reason == ""`

Then the caller can decide when to log? Feels clunky to me to have a logging flag in an interface parameter.

Reply:
For minimizing logging, I wanted to have the log trace logging only, and that's only in the engine. My assumption is that this will be very noisy, and we want it off most of the time, even when we are logging at the debug level.

Reply:
I will rewrite with the interface you suggest, and handle the noisiness higher up the stack.